Sources/SearchResults.swift

```diff
@@ -395,9 +419,10 @@ private func swift2Objc(_ matchLevel: MatchLevel_?) -> MatchLevel {
     /// - parameter content: The JSON content returned by the API.
     /// - parameter disjunctiveFacets: The list of facets to be treated as disjunctive.
     ///
-    @objc public init(content: [String: Any], disjunctiveFacets: [String]) throws {
+    @objc public init(content: [String: Any], disjunctiveFacets: [String], hierarchicalFacets: [String: [String]]) throws {
```

Comment:
This is a change that would break other clients. I would create another constructor with the `hierarchicalFacets` as a parameter and this would just create call it with an empty list